dipy/denoise/localpca.py

```diff
@@ -10,39 +10,86 @@
 from scipy.linalg import eigh
 
 
-def localpca(arr, sigma, mask=None, pca_method='eig', patch_radius=2,
-             tau_factor=2.3, out_dtype=None):
+def _pca_classifier(L, m):
```

Comment:
@RafaelNH would it be better to relocate this function to `noise_estimate.py` ? This will help improve the design from a user standpoint I guess..